gtk/docs.md

```diff
@@ -1 +1,357 @@
 <!-- file * -->
+<!-- struct TextBuffer -->
+You may wish to begin by reading the
+[text widget conceptual overview](https://developer.gnome.org/gtk3/stable/TextWidget.html)
+which gives an overview of all the objects and data
+types related to the text widget and how they work together.
+<!-- impl TextBuffer::fn insert_markup -->
+Inserts the text in `markup` at position `iter`. `markup` will be inserted
+in its entirety and must be nul-terminated and valid UTF-8. Emits the
```

Comment:
Sorry for the delay. I agree that fixing this might would introduce slight inconsistencies but the fact that generated docs aren't perfect shouldn't stop us from making improvements.